lib/signs_ui/config/sign.ex

```diff
@@ -97,49 +102,8 @@ defmodule SignsUi.Config.Sign do
         mode: :static_text,
         line1: config["line1"],
         line2: config["line2"],
-        expires: expiration_from_string(config["expires"])
-      }
-    }
-  end
-
-  @spec from_config(String.t(), map()) :: t()
```

Comment:
The implementation here is identical to `from_json` above it, and has no tests and was only used in one place. I don't really remember how or why it's here... and sort of suspect it might have been a merge conflict or something back in the day? In any case, I removed it, and switched its one caller to `from_json`.